Admin/Extension/PublishableExtension.php

```diff
@@ -41,11 +42,12 @@ public function __construct($formGroup = 'form.group_publish_workflow')
      */
     public function configureFormFields(FormMapper $formMapper)
     {
+        $builder = $formMapper->getFormBuilder()->getFormFactory()->createBuilder(PublishableType::class);
+
         $formMapper
-            ->with($this->formGroup, array('translation_domain' => 'CmfCoreBundle'))
-                ->add('publishable', CheckboxType::class, array(
-                    'required' => false,
-                ))
-            ->end();
+            ->with($this->formGroup, ['translation_domain' => 'CmfCoreBundle'])
+                ->add($builder->get('publishable'))
```

Comment:
In https://github.com/symfony-cmf/core-bundle/pull/207/files#diff-0d9a80ba88f0582b1468194ea97a2000R28 , you do use the `inherit_data` option. Is there a difference when using `->with()` + `->add()` and only using `->add()`? If it is, then we have no choice. Otherwise, I would use `inherit_data` as much as possible.